ConsoleApp6/ColoredGarland.cs

```diff
@@ -0,0 +1,90 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace ConsoleApp6
+{
+    public class ColoredGarland : Lamp
```

Comment:
I don't think the garland can be inherited from Lamp
The garland contains Lamps